app/controllers/subject_rule_effects_controller.rb

```diff
@@ -66,7 +62,7 @@ def update
   rescue Pundit::NotAuthorizedError
     respond_to do |format|
       format.html do
-        flash[:alert] = 'Error updating a subject effect rule. To edit these, please email contact@zooniverse.org with the workflow ID, rule ID, and desired effect details to request these be changes be made by a Zooniverse admin.'
+        flash[:alert] = 'You do not have permission to make this change. Please confirm that you have permissions to the associated subject set or collection.'
```

Comment:
Not blocking - verbose
```suggestion
        flash[:alert] = 'You do not have permission to update this subject rule effect for this project.'
```